test/over_react/component_declaration/component_base_test.dart

```diff
@@ -1240,8 +1445,10 @@ class TestComponentComponent extends UiComponent<TestComponentProps> {
   }
 }
 
+UiFactory<TestComponent2Props> TestComponent2 = ([props]) => TestComponent2Props(props);
```

Comment:
This setup matches that of TestComponent/TestStatefulComponent2; it just was missing here since it wasn't needed